src/javaharness/java/arcs/api/CollectionProxy.java

```diff
@@ -134,7 +135,9 @@ public void store(PortableJson value, String[] keys, String particleId) {
             .emptyObject()
             .put("value", value)
             .put(ModelEntry.KEYS, jsonParser.fromStringArray(Arrays.asList(keys)));
-    port.handleStore(this, (unused) -> {}, data, particleId);
+    port.handleStore(this, new Consumer<PortableJson>() {
+      @Override public void accept(PortableJson unused) {}
+    }, data, particleId);
```

Comment:
Revert this? The old code with the lambda should work, no?

Reply:
this is needed because in the PEC the callbacks are stored in the ThingMapper and these needs to be unique, but all the lambda-s are evaluated as equal.